opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -678,87 +331,34 @@ DenseMatrix transposeDenseMatrix(const DenseMatrix& M)
 
         void prepareFlexibleSolver()
         {
-            // Decide if we should recreate the solver or just do
```

Comment:
Since the scaling stuff (DRS et all) is from your colleagues, I trust you here. Was wondering myself whether this is used/useful.